starter-code/scripts/article.js

```diff
@@ -17,6 +17,9 @@ Article.prototype.toHtml = function() {
   //   or say "(draft)" if it has no publication date:
   this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
   this.publishStatus = this.publishedOn ? 'published ' + this.daysAgo + ' days ago' : '(draft)';
+  var source = $('#article-template').html();
+  var templateRender = Handlebars.compile(source);
+  return templateRender(this);
```

Comment:
👍  Great! Remember that `templateRender` in this case is a custom function that Handlebars returns to us based on the html we initially pass in. From there, we get to invoke it by passing in some context, in our case, `this` is the context referring to the article that invoked our `toHtml()` method.